src/core.jsx

```diff
@@ -59,26 +59,13 @@ export default (
     return class extends Component {
       static displayName = displayName
 
-      state = {
-        props: {},
-        print: true,
-      }
-
-      omitLoadInProps = (props) => {
-        const isLoadAFunction = isFunction(props[loadFunctionName])
+      constructor(props, context) {
+        super(props, context)
 
-        if (isLoadAFunction) {
-          this.setState({
-            props: {
-              ...props,
-              [loadFunctionName]: undefined,
-            },
-          })
-        } else {
-          this.setState({ props })
+        this.state = {
+          props: {},
+          print: true,
         }
-
-        return isLoadAFunction
       }
 
       componentWillMount() {
```

Comment:
You don't have a warning about using the UNSAFE method ?

Reply:
I have, but this is not what I aim in this PR.

I just wanted to upgrade all dependencies before looking into issues :)